src/applications/gi/sass/partials/_gi-compare-drawer.scss

```diff
@@ -173,3 +173,16 @@
     }
   }
 }
+.compare-btn{
+  width: 100% !important;
+}
+.compare-btn::part(button){
+  width: 100% !important;
+}
+.institution {
+  padding: 0em 1em 0em 1em;
```

Comment:
We can use `vads-u-display--flex`, `vads-u-flex-direction--column`, etc in the div's class list instead design.va.gov/foundation/layout/flexbox-grid